### PR TITLE
Add endpoints for proxying payload

### PR DESCRIPTION
**Pull request by xprazak2**
Contains #375 and #325 in a single branch for easier testing. Needs https://github.com/redhataccess/foreman-plugin/pull/56

### CODE REVIEW

File: app/controllers/insights_cloud/api/machine_telemetries_controller.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    end
+
+    private
+
+    def cert_uuid
+      @uuid ||= User.current.login
+    end
+
+    def ensure_org
+      @organization = @host.organization
```
Comment (ShimShtein) [inline app/controllers/insights_cloud/api/machine_telemetries_controller.rb:58]: This line fails for `redhat-access` plugin, when trying to configure the service and press "Check connection" button. The log is:

```
16:28:19 rails.1   | 2020-11-16T16:28:19 [I|app|215ee1cc] Started GET "/redhat_access/r/insights/view/api/v3/me" for 127.0.0.1 at 2020-11-16 16:28:19 +0000
16:28:19 rails.1   | 2020-11-16T16:28:19 [I|app|215ee1cc] Processing by InsightsCloud::Api::MachineTelemetriesController#forward_request as */*
16:28:19 rails.1   | 2020-11-16T16:28:19 [I|app|215ee1cc]   Parameters: {"path"=>"view/api/v3/me"}
16:28:19 rails.1   | 2020-11-16T16:28:19 [W|app|215ee1cc] Action failed
16:28:19 rails.1   | 2020-11-16T16:28:19 [I|app|215ee1cc] Backtrace for 'Action failed' error (NoMethodError): undefined method `organization' for nil:NilClass
16:28:19 rails.1   |  215ee1cc | /home/vagrant/foreman_rh_cloud/app/controllers/insights_cloud/api/machine_telemetries_controller.rb:58:in `ensure_org'
```

The same stack is produced by Insights -> Systems, if you have the username configured.

**Review by ShimShtein — CHANGES_REQUESTED** [review]
Unfortunately it breaks the older plugin. Found a couple of root causes.

**Comment by xprazak2** [comment]
Let's keep the telemetry config for now since it is used in UI. I also added constraints to the routing which should fix the remaining problem.

**Review by ShimShtein — APPROVED** [review]

**Comment by ShimShtein** [comment]
@xprazak2, I took the liberty to rebase your PR, since there were no significant changes.

**Comment by ShimShtein** [comment]
Finally merged! Thanks @xprazak2!

**Pull request merged**